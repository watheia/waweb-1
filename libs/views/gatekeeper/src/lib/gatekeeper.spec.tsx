import { render } from '@testing-library/react';

import Gatekeeper from './gatekeeper';

describe('waweb.views/gatekeeper', () => {
  it('should render successfully', () => {
    const { container } = render(<Gatekeeper onLogin={() => void 0} />);
    expect(container).toBeInstanceOf(HTMLDivElement);
    expect(container).toMatchSnapshot();
  });
});
