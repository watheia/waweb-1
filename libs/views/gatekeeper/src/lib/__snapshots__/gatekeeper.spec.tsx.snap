// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`waweb.views/gatekeeper should render successfully 1`] = `
<div>
  <div
    class="container mx-auto sm:px-6 lg:px-8 root"
  >
    <div
      class="wrapper"
    >
      <h2
        class="appear appear-third show-on-mobile description"
      >
        A tech demo by Watheia Labs.
      </h2>
      <h1
        class="appear appear-third hero"
      >
        Watheia
        <br
          class="show-on-desktop"
        />
         Realtime
      </h1>
      <h2
        class="appear appear-third show-on-tablet description"
      >
        A tech demo by Watheia Labs.
      </h2>
      <div
        class="appear appear-fourth info"
      >
        <p>
          guest
        </p>
        <div
          class="description-separator"
        />
        <p
          class="offline"
        >
          offline
        </p>
      </div>
    </div>
    <form
      class="form appear appear-fifth"
    >
      <div
        class="form-row"
      >
        <label
          class="input-label"
          for="email-input-field"
        >
          <input
            aria-label="Your email address"
            autocomplete="off"
            class="input"
            id="email-input-field"
            placeholder="Enter email to get login url"
            required=""
            type="email"
            value=""
          />
        </label>
        <button
          class="submit login default"
          type="submit"
        >
          Send
        </button>
      </div>
    </form>
    <div
      class="appear appear-fifth root"
    >
      Or login with
       
      <a
        class="link"
        href="/sso"
      >
        Gatekeeper Single Sign-On (SSO)
      </a>
      .
    </div>
  </div>
</div>
`;
